TrkDiag/src/StrawHitDiag_module.cc

```diff
@@ -274,7 +274,7 @@ namespace mu2e
 
   void StrawHitDiag::fillStrawHitDiag(StrawElectronics const& strawele) {
     GeomHandle<DetectorSystem> det;
-    const Tracker& tracker = *GeomHandle<Tracker>();
+    const Tracker& tracker = *GeomHandle<Tracker>(); //FIXME switch to aligned
```

Comment:
There are a number of other downstream diagnostic modules where the digitization change needs to be propagated, ie ComboHitDiag and TrkAna.  Those changes are perhaps outside scope of this PR, but without them, how can you test and study the effects of this PR?

Reply:
mu2e/trkana#125 is ready for when this is merged, makeSD diag and combohitdiag both work. I'll be adding SHD asap